src/frontend/apps/impress/src/features/docs/doc-management/types.tsx

```diff
@@ -27,14 +27,19 @@ export enum LinkReach {
   AUTHENTICATED = 'authenticated',
 }
 
+export enum LinkRole {
+  READER = 'reader',
+  EDITOR = 'editor',
+}
+
 export type Base64 = string;
 
 export interface Doc {
   id: string;
   title: string;
   content: Base64;
   link_reach: LinkReach;
-  link_role: 'reader' | 'editor';
+  link_role: LinkRole;
   accesses: Access[];
   created_at: string;
   updated_at: string;
```

Comment:
I think this type is not only related to this feature, but can be used everywhere. I moved this file to the root of the feature/docs folder. But maybe in another PR? and maybe. And maybe you don't agree with that